e2e/common/misc/pipe_test.go

```diff
@@ -63,7 +63,7 @@ func TestPipe(t *testing.T) {
 					"--name", "throw-error-binding",
 				).Execute()).To(Succeed())
 
-				g.Eventually(IntegrationPodPhase(t, ns, "throw-error-binding"), TestTimeoutMedium).Should(Equal(corev1.PodRunning))
+				g.Eventually(IntegrationPodPhase(t, ns, "throw-error-binding"), TestTimeoutLong).Should(Equal(corev1.PodRunning))
```

Comment:
I moved them to timeout medium on purpose. There's no need to wait 15 minutes to get a failure there. With 5 minutes we give enough time to try the execution.

Reply:
GitHub CI job tends to be slow sometimes and this is why the test is flaky. I checked the failing tests and the integration build is simply not finished after 5 mins sometimes

Reply:
Weird. We should investigate why a simple timer-to-log pipe is taking so long then, I don't think this is happening within the rest of tests, is it?

Reply:
:worried: